src/QMCWaveFunctions/Fermion/MultiSlaterDetTableMethod.h

```diff
@@ -73,6 +73,9 @@ class MultiSlaterDetTableMethod : public WaveFunctionComponent
   template<typename DT>
   using OffloadPinnedAllocator = OMPallocator<DT, PinnedAlignedAllocator<DT>>;
 
+  template<typename DT>
+  using OffloadVector = Vector<DT, OffloadPinnedAllocator<DT>>;
```

Comment:
Since you introduced this type. Please update C_otherDs_ptr_list and det_value_ptr_list to use this new alias.